packages/grafana-plugin-configs/webpack.config.ts

```diff
@@ -28,6 +28,7 @@ const config = async (env: any): Promise<Configuration> => {
       buildDependencies: {
         config: [__filename],
       },
+      cacheDirectory: path.resolve(__dirname, '../../.yarn/.cache/webpack', path.basename(process.cwd())),
```

Comment:
Works great! I think we can also add something similar to the [`EsLintPlugin`](https://webpack.js.org/plugins/eslint-webpack-plugin/#cachelocation) config to prevent the node_modules directory being created in each plugin directory:

```js
cacheLocation: path.resolve(
  __dirname,
  '../../.yarn/.cache/eslint-webpack-plugin',
  path.basename(process.cwd()),
  '.eslintcache'
),
```